### PR TITLE
feat(samples): add media resolution input inside join meeting

### PR DESCRIPTION
**Pull request by rajeshtezu90**
# COMPLETES #<[SPARK-360343](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-360343)>

## This pull request addresses

- Adds media resolution inputs inside join meeting area
- Adds resolution on top right of local and remote video element

## by making the following changes

Screenshots

resolution-input

resolution-on-video

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: docs/samples/browser-plugin-meetings/app.js [hunk 1]
```diff
@@ -1127,9 +1138,44 @@ function clearMediaDeviceList() {
   sourceDevicesVideoInput.innerText = '';
 }
 
+function getLocalMediaSettings() {
+  const meeting = getCurrentMeeting();
```
Comment (arun3528) [inline docs/samples/browser-plugin-meetings/app.js:1142]: We will need to change this to get the setting every 3 second at-least to get the realtime data ,

local and remote resolution will change based on network condition and other parameter so this need to be captured every 3 second ,  can start the interval on media:ready

File: docs/samples/browser-plugin-meetings/app.js [hunk 2]
```diff
@@ -1127,9 +1138,44 @@ function clearMediaDeviceList() {
...
+    return acc;
+  }, '');
+
+  element.innerHTML = optionElements;
+}
+
+(() => {
+  addMediaOptions('local-resolution');
+  addMediaOptions('remote-resolution');
+  addMediaOptions('all-resolution');
```
Comment (arun3528) [inline docs/samples/browser-plugin-meetings/app.js:1176]: lets remove all resolution and setMeetingQuality for meeting itself

**Review by arun3528 — CHANGES_REQUESTED** [review]

File: docs/samples/browser-plugin-meetings/app.js (hunk 1 above)
Comment (rajeshtezu90) [reply docs/samples/browser-plugin-meetings/app.js:1142]: Done

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 3]
```diff
@@ -4078,7 +4078,7 @@ export default class Meeting extends StatelessWebexPlugin {
         LoggerProxy.logger.warn('Meeting:index#getMediaStreams --> Please use `meeting.shareScreen()` to manually start the screen share after successfully joining the meeting');
       }
 
-      audioVideo = {...audioVideo, ...VIDEO_RESOLUTIONS[this.mediaProperties.localQualityLevel]};
+      audioVideo = {...audioVideo, video: {...audioVideo.video, ...VIDEO_RESOLUTIONS[this.mediaProperties.localQualityLevel].video}};
```
Comment (rajeshtezu90) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:4081]: @arun3528 `deviceId` was being removed from video object here.

**Review by Neeraj-swarnkar — APPROVED** [review]
Looks good, Thanks for the changes.

**Pull request merged**